src/command/repo/get.c

```diff
@@ -46,13 +49,81 @@ storageGetProcess(IoWrite *destination)
+
+                    // Encrypted files could be stored directly at the root of the repo if needed
+                    // We should then at least be able to determine the archive or backup directory
+                    StringList *filePathSplitLst = strLstNewSplit(relativeFile, FSLASH_STR);
+                    if (strLstSize(filePathSplitLst) > 1)
+                    {
+                        const String *stanza = strLstGet(filePathSplitLst, 1);
+
+                        // If stanza option is specified, it must match the given file path
+                        if (cfgOptionStrNull(cfgOptStanza) != NULL && ! strEq(stanza, cfgOptionStr(cfgOptStanza)))
+                            THROW_FMT(AssertError, "stanza name '%s' given in option doesn't match the given path", 
+                                strPtr(cfgOptionStr(cfgOptStanza)));
+
+                        if (strEq(strLstGet(filePathSplitLst, 0), STORAGE_PATH_ARCHIVE_STR))
+                        {
+                            // Find the archiveCipherPass
+                            if (! strEndsWithZ(relativeFile, INFO_ARCHIVE_FILE) &&
+                                ! strEndsWithZ(relativeFile, INFO_ARCHIVE_FILE".copy"))
+                            {
+                                InfoArchive *info = infoArchiveLoadFile(
+                                    storageRepo(), strNewFmt(STORAGE_PATH_ARCHIVE "/%s/%s", strPtr(stanza), INFO_ARCHIVE_FILE),
+                                    repoCipherType, cipherPass);
+                                cipherPass = strDup(infoArchiveCipherPass(info));
+                            }
+                            
+                        }
+                        
+                        if (strEq(strLstGet(filePathSplitLst, 0), STORAGE_PATH_BACKUP_STR))
+                        {
+                            if (! strEndsWithZ(relativeFile, INFO_BACKUP_FILE) &&
+                                ! strEndsWithZ(relativeFile, INFO_BACKUP_FILE".copy"))
+                            {
+                                // Find the backupCipherPass
+                                InfoBackup *info = infoBackupLoadFile(
+                                    storageRepo(), strNewFmt(STORAGE_PATH_BACKUP "/%s/%s", strPtr(stanza), INFO_BACKUP_FILE),
+                                    repoCipherType, cipherPass);
+                                cipherPass = strDup(infoBackupCipherPass(info));
+
+                                // Find the manifestCipherPass
+                                if (! strEq(strLstGet(filePathSplitLst, 2), STRDEF("backup.history")) &&
+                                    ! strEndsWithZ(relativeFile, BACKUP_MANIFEST_FILE) &&
+                                    ! strEndsWithZ(relativeFile, BACKUP_MANIFEST_FILE".copy"))
+                                {
+                                   const Manifest *manifest = manifestLoadFile(
```

Comment:
Looks like the indentation here is one space short.

Reply:
fixed, thanks